lib/sablon/html/ast.rb

```diff
@@ -45,23 +65,14 @@ def inspect
     end
 
     class Paragraph < Node
-      attr_accessor :style, :runs
-      def initialize(style, runs)
-        @style, @runs = style, runs
+      attr_accessor :runs
+      def initialize(properties, runs)
+        @properties = properties
+        @runs = runs
       end
 
-      PATTERN = <<-XML.gsub("\n", "")
-<w:p>
-<w:pPr>
-<w:pStyle w:val="%s" />
-%s
-</w:pPr>
-%s
-</w:p>
-XML
-
       def to_docx
-        PATTERN % [style, ppr_docx, runs.to_docx]
+        "<w:p>#{ppr_docx}#{runs.to_docx}</w:p>"
```

Comment:
Following the thought of a `ParagraphProperties` object, this becomes:

```ruby
"<w:p>#{properties.to_docx}#{runs.to_docx}</w:p>"
```

And we can get rid of `ppr_docx`.